src/js/utils/JestUtil.js

```diff
@@ -105,6 +109,32 @@ const JestUtil = {
   },
 
   /**
+   * Mock a different timezone by overriding relevant Date primitive
+   * prototype functions.
+   *
+   * This function will mock Date.getTimezoneOffset and Date.toLocaleString
+   *
+   * @param {String} timezone - The IANA timezone string (ex. Europe/Athens) or 'UTC'
+   */
+  mockTimezone(timezone) {
+    let date = new Date();
+    let timezoneOffset = (
+          new Date(date.toLocaleString(undefined, {timeZone: 'UTC'}))
+        - new Date(date.toLocaleString(undefined, {timeZone: timezone}))
+      ) / 1000 / 60;
+
+    /* eslint-disable no-extend-native */
+    Date.prototype.getTimezoneOffset = function () {
+      return timezoneOffset;
+    };
+    Date.prototype.toLocaleString = function (locale=undefined, options={}) {
```

Comment:
#nit typically we put spaces in between the assignment operator (e.g. `locale = undefined, options = {}`), I'm surprised the linter isn't configured for this. So, I guess you're free to do as you please here!